### DISCUSSION TITLE
Erro de autenticação ao usar o git em linha de comando pt-Br

### DISCUSSION THREAD

**Discussion by connectiva** [community/community#65707] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Após um tempo sem usar o git, fui fazer um comit via linha de comando mas não está aceitando minha senha.
Tenho certeza que está certa.

Preciso habilitar alguma coisa primeiro?

Grato.

**Comment by dnatividade** [comment, marked as answer]
Olá @connectiva 

Você precisa informar agora o token que você gerou no lugar da senha.
